### PR TITLE
Deploy Employment Tribunals ET1 apps into 'perftest' environment

### PR DESCRIPTION
**Pull request by petermoores-version1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-1926

### Change description ###
Deploy Employment Tribunals ET1 Submit your application FE/BE apps into 'perftest' environment.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by robin-ieong — DISMISSED** [review]

File: apps/et/perftest/base/kustomization.yaml
Comment (timja) [inline apps/et/perftest/base/kustomization.yaml:1]: this file will need referencing in https://github.com/hmcts/cnp-flux-config/blob/master/clusters/perftest/base/kustomization.yaml

**Review by timja — CHANGES_REQUESTED** [review]

File: apps/et/perftest/base/kustomization.yaml
Comment (petermoores-version1) [reply apps/et/perftest/base/kustomization.yaml:1]: @timja - Hi Tim, would my latest commit work? Thanks

Comment (petermoores-version1) [reply apps/et/perftest/base/kustomization.yaml:1]: @timja - Hi Tim, gentle nudge to review my latest commit/comment? Thanks

Comment (timja) [reply apps/et/perftest/base/kustomization.yaml:1]: Yes that's good, if you request my review again i get a list of PRs to follow up on :)

image

hit the cycling arrow

**Review by timja — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Comment by petermoores-version1** [comment]
@timja - Thanks for approving this PR, I think two others are required for review automatically as code owners (jordankainos & endakelly). Without those reviewers, I can't merge the PR. I created the PR 22 days ago and really need it merging.

**Review by robin-ieong — APPROVED** [review]

**Pull request merged**
